specs/deliverly_server_SUITE.erl

```diff
@@ -21,7 +23,8 @@ init_per_group(_, Config) ->
   timer:sleep(200),
   Config.
 
-end_per_group(_,Config) ->
+end_per_group(_, Config) ->
+  redis_cli:q(["EVAL \"return redis.call('del', unpack(redis.call('keys', ARGV[1])))\" 0 *"]),
```

Comment:
Line 27 is too long:   redis_cli:q(["EVAL \"return redis.call('del', unpack(redis.call('keys', ARGV[1])))\" 0 *"]),.